controller/validations.go

```diff
@@ -96,6 +96,8 @@ type ValidationsReconciler struct {
 	validationsService *business.IstioValidationsService
 }
 
+var changeMap = map[string]string{}
```

Comment:
This should not be a global var. See comments above about making the changemap internal to `ValidationInfo` but either way we'll need to save some state about the current hash. This should go into the Kiali cache. Not the whole `validationInfo` object but at least the hash, I guess per cluster. `cache.Validations()` could return an object that has the hash(es) in there along with the validations, for example.

Reply:
+1, I knew it had to change, I just wasn't sure what plumbing to do, and so left it easy for the POC.  Before more work I'd like @hhovsepy to run another one of his perf comparisons, to see what's what...

Reply:
rerunning the pref test on this PR

Reply:
@nrfox I've moved this into the cache, let me know if it looks OK to you.